dev/integration_tests/windows_startup_test/lib/main.dart

```diff
@@ -8,6 +8,9 @@ import 'dart:ui' as ui;
 import 'package:flutter/services.dart';
 import 'package:flutter_driver/driver_extension.dart';
 
+// https://docs.microsoft.com/windows/win32/api/dwmapi/ne-dwmapi-dwmwindowattribute
```

Comment:
Add a bit more detail and use a doc comment.

```dart
/// Window attribute that specifies whether a window supports
/// drawing its window decorations in dark mode colors when the
/// dark mode system setting is enabled.
///
/// See: https://docs.microsoft.com/windows/win32/api/dwmapi/ne-dwmapi-dwmwindowattribute
```

Reply:
This was deleted since all Windows logic was moved to C++